frontend/src/components/eMIB/Emib.jsx

```diff
@@ -59,11 +60,16 @@ export const getInboxContent = () => {
     LOCALIZE.emibTest.emails.email6,
     LOCALIZE.emibTest.emails.email7,
     LOCALIZE.emibTest.emails.email8,
-    LOCALIZE.emibTest.emails.email9,
-    LOCALIZE.emibTest.emails.email10
+    LOCALIZE.emibTest.emails.email9
   ];
 };
 
+//Passing Specs as a prop into Inbox causes a bug where the language does not toggle
+// However, inbox's state needs to know the length of the specs to be initialized.
+export const getInboxLen = () => {
```

Comment:
Would it be better to spell it fully to avoid confusion: 
> getInboxLen**gth**

Reply:
fixed in d929685